doc/api/permissions.md

```diff
@@ -482,7 +482,7 @@ flag.
 
 When starting Node.js with `--experimental-permission`,
 the ability to access the file system through the `fs` module, spawn processes,
-use `node:worker_threads` and enable the runtime inspector
+use `node:worker_threads`, native addons and enable the runtime inspector
```

Comment:
On second thought, this doesn't really work grammatically because the list was introduced by "the ability to...". Not a native speaker, but perhaps this works better?

```suggestion
use `node:worker_threads` and native addons, and enable the runtime inspector
```